site/lists/resourcesData.scroll

```diff
@@ -0,0 +1,7 @@
+importOnly
+commaTable
+ series,seriesLink,publisher,year
+ Introduction to the Theory of Programming Languages,https://bertrandmeyer.com/2022/09/28/introduction-theory-programming-languages-full-book-now-freely-available/, Bertrand Meyer,2022
+ Github EbookFoundation,https://github.com/EbookFoundation/free-programming-books/blob/main/books/free-programming-books-subjects.md#computer-science, EbookFoundation
```

Comment:
Only link *directly* to pdfs or single book entries. No links to lists of ebooks. Just links directly to the individual books